### DISCUSSION TITLE
Are there plans to integrate GFN-FF into calculators

### DISCUSSION THREAD

**Discussion by ShangChien** [tblite/tblite#143] [General]
xtb-python is no longer maintained, tblite is the successor, but GFN-FF cannot be used in tblite for the time being.
Hope it will happen, thank you 🐈

**Comment by awvwgk** [comment]
There are now standalone packages for [`gfn0`](https://github.com/pprcht/gfn0) and [`gfnff`](https://github.com/pprcht/gfnff) available, which can be used.
